hie-plugin-api/Haskell/Ide/Engine/PluginsIdeMonads.hs

```diff
@@ -344,14 +344,15 @@ data IdeEnv = IdeEnv
   }
 
 -- | The class of monads that support common IDE functions, namely IdeM/IdeGhcM/IdeDeferM
-class Monad m => MonadIde m where
+class MonadIO m => MonadIde m where
```

Comment:
I guess IO is needed to actually send the message back?

Reply:
Yes